src/lib/components/SelectField/SelectField.jsx

```diff
@@ -69,16 +70,20 @@ export const SelectField = ({
             value={value}
           >
             {
-              options.map((option) => (
-                <option
-                  disabled={option.disabled}
-                  id={id && `${id}__item__${option.value}`}
-                  key={option.value}
-                  value={option.value}
-                >
-                  {option.label}
-                </option>
-              ))
+              options.map((option) => {
+                if ('options' in option) {
+                  return (
+                    <optgroup label={option.label} key={option.label}>
+                      {option.options.map((optgroupOption) => (
+                        <OptionField id={id} key={optgroupOption.label} option={optgroupOption} />
```

Comment:
It is just note and it is up to you whether to change it or not (I don't want you to change every line of code you have written :D). But typically, we try to have at most one prop per line, in case there are more, we try to spread it across more lines, exactly one prop per line. I think we have ESlint rule for that in RACOM projects, but it might not be implemented there. @mbohal 

And when it is not enforced by ESlint, we typically do not comment those things during code review. But it is only way how we can hand over you info about this.